test/functionalTest/cases/2223_default_types_schema_org/default_metadata_types_array_schema_org.test

```diff
@@ -0,0 +1,159 @@
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Default Metadata Types as schema.org
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 09. Create E5/A1/M1, no type, value of A1/M1 is a compound vector
+# 10. GET E5, see that the type of A1/M1 is 'Array' and the type of E5 is 'Thing'
+# 11. Create E6/A1/M1, no type, value of A1/M1 is a compound object
+# 12. GET E6, see that the type of A1/M1 is 'Object' and the type of E6 is 'Thing'
+#
+
+echo "09. Create E5/A1/M1, no type, value of A1/M1 is a compound vector"
+echo "================================================================="
+payload='{
+  "id": "E5",
+  "A1": {
+    "value": "Not Important",
+    "type":  "NotImportant",
+    "metadata": {
+      "M1": {
+        "value": [ 1,2,3 ]
+      }
+    }
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+echo "10. GET E5, see that the type of A1/M1 is 'Array' and the type of E5 is 'Thing'"
+echo "==============================================================================="
+orionCurl --url /v2/entities/E5
+echo
+echo
+
+
+#echo "11. Create E6/A1/M1, no type, value of A1/M1 is a compound object"
+#echo "==========================================================="
+#payload='{
+#  "id": "E6",
+#  "A1": { "a": 1, "b": 2 }
+#}'
+#orionCurl --url '/v2/entities?options=keyValues' --payload "$payload"
+#echo
+#echo
+#
+#
+#echo "12. GET E6, see that the type of A1/M1 is 'Object' and the type of E6 is 'Thing'"
+#echo "============================================================================="
+#orionCurl --url /v2/entities/E6
+#echo
+#echo
+
+
+--REGEXPECT--
+09. Create E5/A1/M1, no type, value of A1/M1 is a compound vector
+=================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E5?type=Thing
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+10. GET E5, see that the type of A1/M1 is 'Array' and the type of E5 is 'Thing'
+===============================================================================
+HTTP/1.1 200 OK
+Content-Length: 140
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "A1": {
+        "metadata": {
+            "M1": {
+                "type": "StructuredValue",
+                "value": [
+                    1,
+                    2,
+                    3
+                ]
+            }
+        },
+        "type": "NotImportant",
+        "value": "Not Important"
+    },
+    "id": "E5",
+    "type": "Thing"
+}
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
+
+
+#11. Create E6/A1/M1, no type, value of A1/M1 is a compound object
```

Comment:
The entire test is funny ... starting with 09, clearly a copy-paste from some other test and then the last two steps commented.
It seems like something I just for debugging only and it got 'git-added' by mistake.

I will take a closer look and remove it if so, or fix it if not.